### ISSUE TITLE
loadedLocales.lang creates a TypeError in SvelteKit

### ISSUE THREAD

**Issue by raremiroir** [ivanhofer/typesafe-i18n#626]
### Version

5.24.2

### Describe the bug

When using loadedLocales[locale][key], the correct string gets printed in the server logs, but the client logs throw an error.

For example: When I log `loadedLocales.en.nav.home.title` to the console, 'Home' gets printed in the server console. but the client console gives this error:
`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'home')`
`    at instance (+page.svelte:38:37)`
`    at init (index.mjs:2016:11)`
`    at new Page (+page.svelte:36:42)`
`    at createProxiedComponent (svelte-hooks.js?v=d474be70:341:9)`
`    at new ProxyComponent (proxy.js?v=d474be70:242:7)`
`    at new Proxy<+page> (proxy.js?v=d474be70:349:11)`
`    at construct_svelte_component_dev (index.mjs:2232:26)`
`    at Array.create_else_block (root.svelte? [sm]:48:83)`
`    at Array.create_default_slot (root.svelte? [sm]:43:22)`
`    at create_slot (index.mjs:71:28)`

Also, the page refuses to load.
Is there any other way to load a string from a specific locale every time in SvelteKit?
Basically, I just want to be able to load a specific string from a specific locale which should show in all locales. So for example an English string which should also show when the current locale is changed to Dutch or French or so.

### Reproduction

1. Create SvelteKit project
2. Install typesafe-i18n
3. Create locales and translations (for example 'nl', 'fr', 'en')
4. `console.log(loadedLocales.nl.some_key.some_string)`
5. Watch how the string gets logged in the server console
6. Watch how the client console errors and the page dies

### Config

```JSON
{
   "baseLocale": "nl",
   "adapter": "svelte",
   "esmImports": true,
   "$schema": "https://unpkg.com/typesafe-i18n@5.24.2/schema/typesafe-i18n.json"
}
```

### Additional information

Running SvelteKit 1.10

**Comment by ivanhofer** [comment]
Hi @raremiroir,
you should not access `loadedLocales` directly. You should always access translations via the `LL` store.

If you need to access all languages at once, you can use the `L` object. Follow the example [here](https://github.com/ivanhofer/typesafe-i18n/blob/main/packages/adapter-node/example/src/i18n/i18n-node.ts) to see how to set it up.

You should probably not load all locales at once inside client code. Use the [`namespaces`](https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/generator#namespaces) feature to only load a subset or even better, create an object with just the data you need and return it inside a `load` function to access it in the client.
